src/main/java/org/kainos/ea/cli/LoginResponse.java

```diff
@@ -0,0 +1,36 @@
+package org.kainos.ea.cli;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class LoginResponse {
+    private String token;
+    private String email;
+
+    public String getToken() {
+        return token;
+    }
+
+    public void setToken(String token) {
+        this.token = token;
+    }
+
+    public String getEmail() {
+        return email;
+    }
+
+    public void setEmail(String username) {
+        this.email = username;
+    }
+
+
+
+    @JsonCreator
+    public LoginResponse(
```

Comment:
The token already contains the email address in it so, you don't need to return the email address and token in the response. That also means you can get rid of all the immutable pair stuff